### PR TITLE
Add GodotVR respository info to ARVR Primer

### PR DESCRIPTION
**Pull request by skyace65**
Adds information on the official GodotVR repository to the ARVR Primer page. It now lists all the official plugins and demo projects with links to them. I've also altered the small paragraph at the end of the nodes section because it implied that this is all you need to run VR in general, when it's all you need if you're using the native mobile interface. I also updated the screenshot there to one with a script attached so people don't think you can run a VR scene without a script.

The reason everything looks like it's been deleted and re-add is because when this page was made paragraphs were just one or two extremely long lines.

On a related note. We should really change the author/submitter of the official VR plugins in the asset library to Godot.

### CODE REVIEW

File: tutorials/vr/vr_primer.rst [hunk 1]
```diff
@@ -32,52 +45,134 @@ To enable an interface, you execute the following code:
...
+  the Oculus Go and Oculus Quest. The Quest will require additional setup documented `here <doc_developing_for_oculus_quest>`_.
+* `Godot OpenVR <https://github.com/GodotVR/godot_openvr>`_ (not to be confused with OpenXR)
+  supports the OpenVR SDK used by Steam.
+* `Godot Oculus <https://github.com/GodotVR/godot_oculus>`_ supports the Oculus SDK
+  (desktop headsets only).
+* `Godot OpenHMD <https://github.com/GodotVR/godot_openhmd>`_ supports OpenHMD, an open source
+  API and drivers for headsets.
 
-There are a few other subjects that we need to briefly touch upon in this primer that are important to know.
+These plugins can be downloaded from github or the Godot Asset Library. The author of them will
```
Comment (akien-mga) [inline tutorials/vr/vr_primer.rst:131]: "The author of them" sounds awkward, I'd use "Their author".

But I'd also prefer to simply drop the sentence, I don't think it's relevant information for the docs.

Comment (akien-mga) [inline tutorials/vr/vr_primer.rst:131]: Make sure to capitalize **G**it**H**ub properly.

**Comment by akien-mga** [comment]
While I prefer text hard wrapped at 80 chars, it makes changes difficult to review as we can't know at a glance what's the new content.

File: tutorials/vr/vr_primer.rst [hunk 2]
```diff
@@ -32,52 +45,134 @@ To enable an interface, you execute the following code:
...
+a lot more than most people realize. For mobile VR, you have to be extra careful here, but even for
+desktop games, there are three factors that make life extra difficult:
+
+* You are rendering stereoscopic, two for the price of one. While not exactly doubling the work load
+  and with things in the pipeline such as supporting the new MultiView OpenGL extension in mind, there
+  still is an extra workload in rendering images for both eyes
+* A normal game will run acceptably on 30fps and ideally manages 60fps. That gives you a big range to
+  play with between lower end and higher end hardware. For any HMD application of AR or VR, however,
+  60fps is the absolute minimum and you should target your games to run at a stable 90fps to ensure your
+  users don't get motion sickness right off the bat.
```
Comment (Calinou) [inline tutorials/vr/vr_primer.rst:169]:
```suggestion
* A normal game will run acceptably on 30 FPS and ideally manages 60 FPS. That gives you a big range to
  play with between lower end and higher end hardware. For any HMD application of AR or VR, however,
  60 FPS is the absolute minimum and you should target your games to run at a stable 90 FPS to ensure your
  users don't get motion sickness right off the bat.
```

File: tutorials/vr/vr_primer.rst [hunk 3]
```diff
@@ -32,52 +45,134 @@ To enable an interface, you execute the following code:
...
+ARVROrigin node called world scale. For instance, setting this to a value of 10 changes our coordinate
+system so 10 units = 1 meter.
+
+Performance is another thing that needs to be carefully considered. Especially VR taxes your game
+a lot more than most people realize. For mobile VR, you have to be extra careful here, but even for
+desktop games, there are three factors that make life extra difficult:
+
+* You are rendering stereoscopic, two for the price of one. While not exactly doubling the work load
+  and with things in the pipeline such as supporting the new MultiView OpenGL extension in mind, there
+  still is an extra workload in rendering images for both eyes
```
Comment (Calinou) [inline tutorials/vr/vr_primer.rst:165]:
```suggestion
  still is an extra workload in rendering images for both eyes.
```

File: tutorials/vr/vr_primer.rst [hunk 4]
```diff
@@ -32,52 +45,134 @@ To enable an interface, you execute the following code:
...
+* A normal game will run acceptably on 30fps and ideally manages 60fps. That gives you a big range to
+  play with between lower end and higher end hardware. For any HMD application of AR or VR, however,
+  60fps is the absolute minimum and you should target your games to run at a stable 90fps to ensure your
+  users don't get motion sickness right off the bat.
+* The high FOV and related lens distortion effect require many VR experiences to render at double the
+  resolution. Yes a VIVE may only have a resolution of 1080x1200 per eye, we're rendering each eye at
```
Comment (Calinou) [inline tutorials/vr/vr_primer.rst:171]:
```suggestion
  resolution. A HTC Vive may only have a resolution of 1080x1200 per eye, but we're rendering each eye at
```

File: tutorials/vr/vr_primer.rst [hunk 5]
```diff
@@ -32,52 +45,134 @@ To enable an interface, you execute the following code:
...
+  2160x2400 as a result. This is less of an issue for most AR applications.
+
+All in all, the workload your GPU has in comparison with a normal 3D game is a fair amount
+higher. While things are in the pipeline to improve this, such as MultiView and foveated rendering,
+these aren't supported on all devices. This is why you see many VR games using a more art style
```
Comment (Calinou) [inline tutorials/vr/vr_primer.rst:176]:
```suggestion
these aren't supported on all devices. This is why you see many VR games using a less realistic art style
```

**Comment by Calinou** [comment]
Nevermind the comment above, I thought those were new paragraphs.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**